10.0/galera/run.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+if [ "$NODE_IP" ]; then
+	echo "wsrep_node_address=$NODE_IP" >> /etc/mysql/conf.d/galera.cnf
+fi
+
+if [ "$MYSQL_CLUSTER_NAME" ]; then
+	echo"wsrep_cluster_name=$MYSQL_CLUSTER_NAME" >> /etc/mysql/conf.d/galera.cnf
+fi
+
+if [ "$CLUSTER_ADDRESS" ]; then
+	export MYSQL_ALLOW_EMPTY_PASSWORD="yes"
+fi
+
+echo "wsrep_cluster_address=${CLUSTER_ADDRESS-gcomm://}" >> /etc/mysql/conf.d/galera.cnf
+
+sed '/mysql_tzinfo_to_sql/d' -i /docker-entrypoint.sh
```

Comment:
What's this about? It seems like it may be better to update the underlying image to have a way to stop the entrypoint from running the associated code instead of having the derived image delete code from the parent image's entrypoint.